tmt/steps/prepare/feature/epel-enable.yaml

```diff
@@ -54,11 +63,13 @@
             name: yum-utils
             state: present
 
-        - name: Enable repos of 'epel'
-          shell: yum-config-manager --enable epel epel-debuginfo epel-source
+        - name: Enable EPEL repos
+          ansible.builtin.command: yum-config-manager --enable epel epel-debuginfo epel-source
+          register: output
+          changed_when: output.rc != 0
 
-    # CentOS Stream N where N >= 8
-    - when: ansible_distribution == "CentOS" and ansible_distribution_major_version|int >= 8
+    - name: Enable EPEL and EPEL-Next repos on CentOS Stream 8+
+      when: ansible_distribution == "CentOS" and ansible_distribution_major_version | int >= 8
```

Comment:
Looking at the above "Disable EPEL" - `when: (ansible_distribution == "RedHat" or ansible_distribution == "CentOS")` maybe we could do the same for "Enable EPEL", as it looks like it's doing the same things for RHEL and CentOS Stream?

Reply:
I meant merging these two
ansible_distribution == "RedHat" and ansible_distribution_major_version|int >= 8
ansible_distribution == "CentOS" and ansible_distribution_major_version|int >= 8

into one
(ansible_distribution == "RedHat" or ansible_distribution == "CentOS") and ansible_distribution_major_version|int >= 8

but I didn't realize there is a different workflow (RHEL downloading rpm from fedoraproject.org), sorry for the noise.

I think it's missing `dnf config-manager --set-enabled crb`, but regardless, not related to this PR.